### PR TITLE
Update sc-google-code-style-check.yml

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

File: .github/workflows/sc-google-code-style-check.yml [hunk 1]
```diff
@@ -10,4 +10,4 @@ jobs:
       - uses: actions/checkout@v2 # v2 minimum required
       - uses: axel-op/googlejavaformat-action@v2.0.0
         with:
-          args: "--replace"
\ No newline at end of file
+          args: "--dry-run --set-exit-if-changed"
```
Comment (oliver-walker-hms) [inline .github/workflows/sc-google-code-style-check.yml]: maybe add more info in the commit body? I'm not sure whats going on with this

Comment (alexjhawk) [reply .github/workflows/sc-google-code-style-check.yml]: added

**Review by TomKimsey — DISMISSED** [review]

**Review by oliver-walker-hms — DISMISSED** [review]

File: src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java [hunk 2]
```diff
@@ -1,15 +1,7 @@
 package com.hms_networks.americas.sc.historicaldata;
 
 import com.hms_networks.americas.sc.fileutils.FileAccessManager;
-import java.io.BufferedReader;
```
Comment (TomKimsey) [inline src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java:4]: 90% of the time you have "and" in a commit title it should be two commits.

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java (hunk 2 above)
Comment (alexjhawk) [reply src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java:4]: Ill change that. Also, the google code style file changed again. I configured it to skip checking the import order. It now will only throw an error for unused imports

**Comment by alexjhawk** [comment]
removed a sneaky whitespace change

**Review by TomKimsey — APPROVED** [review]

**Pull request merged**